microcosm_pubsub/caching/resource_cache.py

```diff
@@ -0,0 +1,27 @@
+from microcosm.api import defaults, typed
+from microcosm.config.types import boolean
+from microcosm_pubsub.caching.memcached import MemcachedCache
+
+
+@defaults(
+    enabled=typed(boolean, default_value=True),
+    host="localhost",
+    port=typed(int, default_value=11211),
+)
```

Comment:
We want to auto-cache only some whitelisted resources - not everything (maybe we can user regex?, schema def?).
This is a good place to configure that

Reply:
@pelegdvir good catch - was meaning to do something like that e.g check for `Event` / `_event` but forgot to include - will add this in as some `whitelist_mask` parameter or so